src/resources/i18n/translations/en.json

```diff
@@ -180,30 +181,30 @@
   "Auto": "Auto",
   "Auto Downloading Off": "Auto-downloading off",
   "Auto Downloading On": "Auto-downloading on",
-  "Auto queue position helper text": "Set where new episodes should be added to your queue when auto-queue is enabled for a podcast.",
-  "Automatically add new episodes to queue": "Automatically add new episodes to queue",
+  "Auto queue position helper text": "Only has an impact if auto-queueing is turned on for a podcast.",
+  "Automatically add new episodes to queue": "Auto-enqueue new episodes",
   "Autoscroll Off": "Auto-scroll off",
   "Autoscroll On": "Auto-scroll on",
   "Back": "Back",
   "Back To App": "Back To App",
   "Back To Login": "Back To Login",
   "Balance": "Balance",
-  "Bitcoin Wallet": "Bitcoin Wallet",
+  "Bitcoin Wallet": "Bitcoin wallet",
   "Boost": "Boost",
   "Boost Amount": "Boost amount",
   "Boost Amount for this Podcast": "Boost amount for this podcast",
   "Boost Sent": "Boost sent",
   "Boost splits": "Boost splits",
   "brandName chat is powered by": "Podverse chat uses these libre libraries",
   "Bug Report": "Report a bug",
-  "Buy a Podverse premium membership": "Buy Podverse Premium for someone",
+  "Buy a Podverse premium membership": "Gift Podverse Premium to someone",
```

Comment:
Ah I might have missed this change before, but this base value is wrong. "Buy Podverse Premium for someone" doesn't make much sense, because you're only buying the membership for yourself. You can only sign-up with your own email, and we don't have a way to transfer memberships to other people.